api/v1/views/index.py

```diff
@@ -5,8 +5,27 @@
 
 from api.v1.views import app_views
 from flask import Flask, jsonify
+from models import storage
 
 
-@app_views.route('/status')
+@app_views.route('/status', strict_slashes=False)
+
 def jsonStatus():
     return jsonify(status=' ok')
+
+
+@app_views.route('/stats', methods=['GET'], strict_slashes=False)
+def stats():
+    """ retrieves number of objects by type """
+    classes = {
```

Comment:
Small change, but I would leave the values in the dictionary as either blank strings ("") or as a zero ("0"). because it makes it a little clearer that the point of the dictionary is to hold the counts.
Also, probably would rename it from classes to countOfEachClassType.
Just removes the ambiguity of what the point of the dictionary is and makes the code more readable.